src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -318,6 +319,12 @@ class TrialWaveFunction : public MPIObjectBase
                            std::vector<ValueType>& dhpsioverpsi,
                            bool project = false);
 
+  void flex_evaluateParameterDerivatives(const RefVector<TrialWaveFunction>& wf_list,
```

Comment:
Please make this static like other flex_ functions.

Reply:
Fixed.